main.java

```diff
@@ -11,64 +13,99 @@ public static void main(String[] args) {
 
         System.out.println("Choose your magical creature to use for battle!");
         System.out.println("-".repeat(93));
-        //list of creatures
-        System.out.println("Troll");
-        System.out.println("Dragon");
-        System.out.println("Hippogriff");
-        System.out.println("Unicorn");
-        System.out.println("Cerberus");
-        System.out.println("Mandrake");
+         
+     //creating the creatures 
+        var myDragon = new Dragon();
+        var myHippogriff = new Hippogriff();
+        var myMandrake = new Mandrake();
+        var myThreeHeadedDog = new ThreeHeadedDog();
+        var myTroll = new Troll();
+        var myUnicorn = new Unicorn();
+
+    //make array of creatures to number
+      ArrayList<String> Creat = new ArrayList<String>();  
```

Comment:
That's the clunky way of doing it.  You can have an array of creatures instead.  Each item in the array has an implicit index associated with it.  You can use that index to display to the user and retrieve the correct creature from the array.  Then you don't have keep this unmaintainable association of strings to index to creature.
```suggestion
      ArrayList<Creature> availableCreatures = new ArrayList<Creature>();  
      availableCreatures.add(myDragon);
      availableCreatures.add(myHippogriff);
      // etc.
```